OasisDEX/oasis-borrow
# feat: remove team members

# [Remove Team Members](https://app.shortcut.com/oazo-apps/story/12081/update-https-summer-fi-about-remove-leavers)

## Changes 👷‍♀️
- Removed Sam and Adam pictures from the static content. This effectively removes them from the `About` page
  
## How to test 🧪
Run the app, go to the `About->Team` section and confirm that neither Sam nor Adam are there
